src/test/java/be/syntra/devshop/DevshopBack/controllers/ProductControllerTest.java

```diff
@@ -296,4 +307,88 @@ void canUpdateCategoryTest() throws Exception {
 
         verify(categoryService, times(1)).updateCategory(any(), any());
     }
+
+    @Test
+    @WithMockUser
+    void canGetStarRatingFromUserTest() throws Exception {
+        // given
+        final Product dummyProduct = createNonArchivedProduct();
+        final StarRating rating = createRating();
+        final StarRatingDto ratingDto = createRatingDto();
+        when(ratingService.getRatingFromUser(dummyProduct.getId(), rating.getUserName())).thenReturn(rating);
+        when(starRatingMapper.mapToDto(rating)).thenReturn(ratingDto);
+
+        // when
+        ResultActions resultActions =
+                mockMvc.perform(
+                        get("/products/" + rating.getUserName() + "/ratings/" + dummyProduct.getId())
+                                .contentType(MediaType.APPLICATION_JSON)
+                                .content(jsonUtils.asJsonString(ratingDto)));
```

Comment:
adding payload to the call isn't needed, only thing the method expects is the id

Reply:
fixed